package.json

```diff
@@ -25,7 +25,6 @@
     "jquery": "^3.5.0",
     "mongoose": "^5.9.11",
     "nodemon": "^2.0.3",
-    "puppeteer": "^3.0.2",
```

Comment:
i understand it takes up a lot of space but its only downloaded once, no?

Reply:
i removed it because it kept failing in the npm install on my ec2 and im not using it anyway haha